### DISCUSSION TITLE
enquiry

### DISCUSSION THREAD

**Discussion by jesutobi** [Open-Science-Community-Saudi-Arabia/Open-Science-Community-in-Saudi#5] [Outreachy 2022] [answered]
@alswajiab
@BatoolMM
Hello my name name is Jesutobi , i am an outreachy intern and a frontend developer that wants to contribute to this project ,but when i tried to fill the form to join the community i got a notification that the form has exceeded quota,with should i do next

**Comment by BatoolMM** [comment, marked as answer]
Hi @jesutobi, 
Apologies for the broken link, is [this one](https://form.jotform.com/221813832742353) working for you?

**Reply by jesutobi** [reply]
>@BatoolMM I just tried the link you dropped, I have successfully filled the form, thank you...... What do I need to do next, I can't wait to get my fingers busy

**Reply by BatoolMM** [reply]
We have two projects for Outreachy, you can find them here:
1- [Improve the bilinguality and accessibility of the website](https://github.com/Open-Science-Community-Saudi-Arabia/OSCSA_Website/issues/12)
2- Improve Localisation for [Open Educational Resources](https://github.com/Open-Science-Community-Saudi-Arabia/localisation/issues/13).

Feel free to comment under the issue, if you have any questions!

**Reply by jesutobi** [reply]
I would, thank you @BatoolMM
